data/transactions/logic/evalStateful_test.go

```diff
@@ -157,14 +152,11 @@ func (l *testLedger) AssetHolding(addr basics.Address, assetID basics.AssetIndex
 	return basics.AssetHolding{}, fmt.Errorf("no such address")
 }
 
-func (l *testLedger) AssetParams(addr basics.Address, assetID basics.AssetIndex) (basics.AssetParams, error) {
-	if br, ok := l.balances[addr]; ok {
-		if asset, ok := br.assets[uint64(assetID)]; ok {
-			return asset, nil
-		}
-		return basics.AssetParams{}, fmt.Errorf("No asset for account")
+func (l *testLedger) AssetParams(assetID basics.AssetIndex) (basics.AssetParams, error) {
+	if asset, ok := l.assets[assetID]; ok {
+		return asset, nil
 	}
-	return basics.AssetParams{}, fmt.Errorf("no such address")
+	return basics.AssetParams{}, fmt.Errorf("No such asset")
```

Comment:
see https://github.com/golang/go/wiki/CodeReviewComments#error-strings : 
```
Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context. That is, use fmt.Errorf("something bad") not fmt.Errorf("Something bad"), so that log.Printf("Reading %s: %v", filename, err) formats without a spurious capital letter mid-message. This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.```

Reply:
fixed